__tests__/discord-groups/discord-group.test.js

```diff
@@ -133,7 +133,7 @@ describe('Discord Groups Page', () => {
       '.group-role',
       (list) => list.length,
     );
-    expect(groupListLength).toBe(1);
+    expect(groupListLength).toBe(3);
```

Comment:
why not 1, why 3?

Reply:
@Pratiyushkumar Before there was only one group details in mockdata. Later to test search function i added 2 more group details to it. So, I changed the length to be 3

Reply:
@Pratiyushkumar I have used debounce technique for optimization.